include/ConditionManager.h

```diff
@@ -89,12 +95,14 @@ class Setup {
     private:
 
         std::atomic<int> counter;
-        std::atomic<int> m_hv;
 
         std::mutex m_mtx;
 
         std::atomic<State> m_state;
 
         std::thread thread_handle_HV;
         std::thread thread_handle_TDC;
+
+        std::vector<int> m_hvpmt_setValues;
+        std::vector<int> m_hvpmt_setStates;
```

Comment:
For now it's very good but at some point we might want to switch to something like:
```
struct HVMPT {
    setValues,
    setStates
} m_hvpmt;
```